app/src/main/java/org/dhis2/utils/filters/workingLists/WorkingListChipGroup.kt

```diff
@@ -0,0 +1,32 @@
+package org.dhis2.utils.filters.workingLists
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import com.google.android.material.chip.ChipGroup
+import org.dhis2.databinding.ItemFilterWorkingListChipBinding
+
+class WorkingListChipGroup @JvmOverloads constructor(
```

Comment:
Not that important, but can we add View as suffix to the class name to have a better understanding of what this class is about?